ui/cypress/support/index.js

```diff
@@ -0,0 +1,23 @@
+// ***********************************************************
```

Comment:
we should better stick to typescript, the filename should be rather changed to index.ts and adjusted accordingly

Reply:
File extension changed to .ts and its content adjusted accordingly.